src/project-tests/product-landing-page-tests.js

```diff
@@ -121,10 +121,19 @@ export default function createProductLandingPageTests() {
           video.nodeName === 'VIDEO' || video.nodeName === 'IFRAME',
           '#video should be an <iframe> or <video> element '
         );
+        // To accommodate `<source>` elements within `<video>` elements
+        // https://developer.mozilla.org/en-US/docs/Web/HTML/Element/source
+        const sourceNode = video.hasChildNodes();
```

Comment:
This might need to be more robust. A `<video>` element can also have `<track>` children. And `<track>` elements can have a `src` attribute.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/track

Reply:
check updated please.